features/attach_validtoken.feature

```diff
@@ -494,3 +494,65 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
           | bionic  | disabled |
           | focal   | n/a      |
           | jammy   | n/a      |
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attach and Check for contract change in status checking
+       Given a `<release>` machine with ubuntu-advantage-tools installed
+       When I attach `contract_token` with sudo
+       Then stdout matches regexp:
+       """
+       UA Infra: ESM enabled
+       """
+       And stdout matches regexp:
+       """
+       This machine is now attached to
+       """
+       And stdout matches regexp:
+       """
+       esm-infra    +yes      +enabled  +UA Infra: Extended Security Maintenance \(ESM\)
+       """
+       When I create the file `/tmp/machine-token-overlay.json` with the following:
+       """
+       {
+           "machineTokenInfo": {
+               "contractInfo": {
+                   "resourceEntitlements": [
+                       {
+                           "type": "esm-apps",
+                           "entitled": false
+                       }
+                   ]
+               }
```

Comment:
Let's just change this to the use-case that triggered the need for this feature
```suggestion
               "contractInfo": {
                   "effectiveTo": "2000-01-02T03:04:05Z"
               }
```

Reply:
Make sense, let me add.